trolldb/config/config.py

```diff
@@ -104,6 +108,7 @@ class AppConfig(BaseModel):
     subscriber: SubscriberConfig
 
 
+@validate_call
```

Comment:
Can we remove this and type hinting? we loose in genericity otherwise.
I this function, we use `open` which works with a lot of types...

Reply:
I removed the `@validate_call` decorator. However, I modified the type hint so that it now matches the type hint for the `open()` function. As a result, some more error handling + logging has been also added,

Reply:
Done!